commit 7f513ec8b3cd7816a897052bd472c0bfd553e910
Author: Roland McGrath <mcgrathr@chromium.org>
Date:   Thu Sep 8 09:37:42 2011 -0700

    Hurd: fix use of uninitialized value in fork port copying.

---
 sysdeps/mach/hurd/fork.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/sysdeps/mach/hurd/fork.c
+++ b/sysdeps/mach/hurd/fork.c
@@ -1,4 +1,4 @@
-/* Copyright (C) 1994,1995,1996,1997,1999,2001,2002,2004,2005,2006
+/* Copyright (C) 1994,1995,1996,1997,1999,2001,2002,2004,2005,2006,2011
 	Free Software Foundation, Inc.
    This file is part of the GNU C Library.
 
@@ -313,7 +313,7 @@
 	    {
 	      /* This is a send right or a dead name.
 		 Give the child as many references for it as we have.  */
-	      mach_port_urefs_t refs, *record_refs = NULL;
+	      mach_port_urefs_t refs = 0, *record_refs = NULL;
 	      mach_port_t insert;
 	      mach_msg_type_name_t insert_type = MACH_MSG_TYPE_COPY_SEND;
 	      if (portnames[i] == newtask || portnames[i] == newproc)
